scripts/modelbench/entrance.sh

```diff
@@ -21,7 +21,7 @@ CHANNELS=${15:-first}
 WRAPPER=${16:-default}
 HF_TOKEN=${17:-hf_xx}
 BACKEND=${18:-inductor}
-SUITE=${19:-torchbench}
+SUITE=${19:-all}
```

Comment:
it should not impact the guilty commit search part, right?

Reply:
Yes, cuz guilty commit job set torchbench as it's default value in Jenkins job parameters already.